### ISSUE TITLE
Minimap makes scrollbar hard to notice

### ISSUE THREAD

**Issue by sm-g** [ForkIssues/TrackerWin#471]
![Fork_2019-10-04_21-35-24](https://user-images.githubusercontent.com/7280273/66231359-0d867a80-e6ef-11e9-96eb-ecaa5faa24e4.png)
![Fork_2019-10-04_21-35-40](https://user-images.githubusercontent.com/7280273/66231363-10816b00-e6ef-11e9-91e1-557cccda9112.png)

also there is bug - minimap disappears after switching theme

**Comment by DanPristupov** [comment]
We made the scrollbar wider in 1.41. After 3 weeks of use, I think the scrollbar looks ok now.

**Comment by Stanzilla** [comment]
Just mentioned this over at https://github.com/ForkIssues/TrackerWin/issues/607

**Comment by stijnherreman** [comment]
@DanPristupov was that change reverted? When using 1.46.1.0 the scrollbar is as wide as in the OP.

**Comment by DanPristupov** [comment]
@stijnherreman No, we increased the width and did not revert that. However the scrollbars are still not as wide as in Windows Explorer, for example.

**Comment by ddbtrmatic** [comment]
I also find the scrollbar handle to be very difficult to see. I'm sitting about 18" away from a 27" 1440p monitor.

An option to separate or disable the diff map in the scrollbar, or a more pronounced contrast would be greatly appreciated.

This is particularly pronounced on very large diffs (extreme example - package.lock, 20K+ lines).

![image](https://user-images.githubusercontent.com/117312398/213526225-49c34266-eb6e-4d5b-917a-d0e39e137fc8.png)

Compared to other editors:

vscode

![image](https://user-images.githubusercontent.com/117312398/213526652-90d9c739-bcc3-4155-ad34-4cca18542c72.png)

visual studio

![image](https://user-images.githubusercontent.com/117312398/213526819-00d66677-1540-4aa0-b725-1310f36e5fe8.png)
